src/mutations/messages.mutation.ts

```diff
@@ -74,6 +74,12 @@ export const DELETE_MULTIPLE_MESSAGE_MUTATION = gql`
   }
 `;
 
+export const DELETE_ALL_MESSAGE_MUTATION = gql`
+  mutation deleteAllMessages {
+    deleteAllMessages
```

Comment:
doesn't the mutation need parenthesis? like if you are calling a function, `deleteAllMessages()`?